app/views/add_confirmation.scala.html

```diff
@@ -0,0 +1,48 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.i18n.Messages
+@import play.api.mvc.Request
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.html.Layout
+@import views.html.fragments.Heading1
+@this(
+        govukSummaryList: GovukSummaryList,
+        h1: Heading1,
+        mainLayout: Layout
+)
+
+@(entry: Entry, protectedUserRecord: ProtectedUserRecord)(implicit request: Request[_], messages: Messages)
+
+@mainLayout(
+    pageTitle = Messages("add.confirm.title"),
+    backLink = Some(BackLink(href = controllers.routes.SiProtectedUserController.homepage.url))
+) {
+    @h1("add.confirm.header")
+    @govukSummaryList(
+        SummaryList(
+            Seq(
+                SummaryListRow(key = Key(Text(Messages("add.confirm.entryId"))), value = Value(content = Text(protectedUserRecord.entryId))),
+                SummaryListRow(key = Key(Text(Messages("add.confirm.status"))), value = Value(content = Text(Messages(s"add.confirm.action.${entry.action}")))),
+                SummaryListRow(key = Key(Text(protectedUserRecord.body.taxId.name.toString)), value = Value(content = Text(protectedUserRecord.body.taxId.value))),
+                SummaryListRow(key = Key(Text(Messages("add.confirm.identityProvider"))), value = Value(content = Text(protectedUserRecord.body.identityProviderId.map(_.name).getOrElse("")))),
+                SummaryListRow(key = Key(Text(Messages("add.confirm.identityProviderId"))), value = Value(content = Text(protectedUserRecord.body.identityProviderId.map(_.value).getOrElse("")))),
+                SummaryListRow(key = Key(Text(Messages("add.confirm.group"))), value = Value(content = Text(protectedUserRecord.body.group))),
+                SummaryListRow(key = Key(Text(Messages("add.confirm.addedByTeam"))), value = Value(content = Text(protectedUserRecord.body.addedByTeam.getOrElse("")))),
+            )
```

Comment:
Do we need to show updatedByTeam or any of the user fields here?

Reply:
wasnt in the wireframes so I guess not